Lib/idlelib/idle_test/test_configdialog.py

```diff
@@ -91,7 +94,7 @@ def test_set_samples_bold_toggle(self):
         d.set_samples()
         self.assertTrue(d.font_sample == d.highlight_sample == expected1)
 
-        # Test bold_toggle.
+        # Test bold_toggle.  If this fails, problem precedes set_samples.
         d.bold_toggle.invoke()
         self.assertFalse(d.font_bold.get())
         self.assertTrue(d.font_sample == d.highlight_sample == expected0)
```

Comment:
I know there's a good answer for my question, but I was also wondering why the call to var_changed_font isn't being checked by looking at changes?  You mentioned the cycle and the calls in the previous post and, since the trace functions are being invoked, do they need to be checked?  Or should there be 1 test class for all the trace callbacks?  I guess I was thinking about the open issues for creating a Variable class and how that would be tested now and then when the code is changed.